hive/r/1-hive-create-table.R

```diff
@@ -0,0 +1,34 @@
+
+source("r/0-configure.R")
```

Comment:
Please change the scripts so the working directory is the project root. This will be consistent with the other scripts in this project.